src/ifvc.c

```diff
@@ -268,6 +270,20 @@ void buildIfVc(void) {
             mask = ((struct sockaddr_in *)&IfReq.ifr_addr)->sin_addr.s_addr;
             subnet = addr & mask;
 
+            dp = getIfByName(IfPt->ifr_name, 1);
+            if (dp != NULL && dp->allowednets != NULL) {
+                allowednet = (struct SubnetList *)malloc(sizeof(struct SubnetList));
+                if (allowednet == NULL) my_log(LOG_ERR, 0, "Out of memory !");
```

Comment:
I believe code above avoids NULL pointer dereference because logging an error via 
>`if (allowednet == NULL) my_log(LOG_ERR, 0, "Out of memory !");`

leads to 'src/syslog.c' and there it exits after logging:

> `    if( Severity <= LOG_ERR )
>         exit( -1 );
> `